torch/distributed/elastic/agent/server/api.py

```diff
@@ -356,37 +357,6 @@ def is_failed(self) -> bool:
         return self.state == WorkerState.FAILED
 
 
-def _get_socket_with_port() -> socket.socket:
```

Comment:
Hi @kurman. After delete '_get_socket_with_port' api. Can I know what is the equivalent API/method?

Reply:
@Yejing-Lai sorry missed your note. Would `from torch.distributed.elastic.utils.distributed import get_free_port` work? rendezvous/api.py uses the function now.